apps/docs/docs/reference/ui/atoms/Button.mdx

```diff
@@ -54,8 +58,23 @@ import { Button } from '@faststore/ui'
 
 ### Do's
 
+- Use standard button designs and shapes to make your button **look clickable**.
+- Use styles to **communicate the importance of an action** by visually differentiating Primary and Secondary buttons.
+- Make buttons **finger-friendly** for mobile users and large enough for **reliable interactions**.
```

Comment:
What about suggesting the minimum `44px` tap size?
https://www.w3.org/WAI/WCAG21/quickref/#motion-actuation